app/mailers/school_mailer.rb

```diff
@@ -30,4 +32,58 @@ def nomination_confirmation_email(user:, school:, start_url:)
       },
     )
   end
+
+  def school_partnership_notification_email(recipient:, provider_name:, cohort:, nominate_url:, challenge_url:)
+    template_mail(
+      SCHOOL_PARTNERSHIP_NOTIFICATION_EMAIL_TEMPLATE,
+      to: recipient,
+      rails_mailer: mailer_name,
+      rails_mail_template: action_name,
+      personalisation: {
+        provider_name: provider_name,
+        cohort: cohort,
+        nominate_url: nominate_url,
+        challenge_url: challenge_url,
+      },
+    )
+  end
+
+  def coordinator_partnership_notification_email(recipient:, provider_name:, cohort:, start_url:, challenge_url:)
+    template_mail(
+      COORDINATOR_PARTNERSHIP_NOTIFICATION_EMAIL_TEMPLATE,
+      to: recipient,
+      rails_mailer: mailer_name,
+      rails_mail_template: action_name,
+      personalisation: {
+        provider_name: provider_name,
+        cohort: cohort,
+        start_url: start_url,
+        challenge_url: challenge_url,
+      },
+    )
+  end
+
+  def self.send_school_partnership_notification_email(recipient:, provider_name:, cohort:, nominate_url:, challenge_url:)
```

Comment:
I'd put these 2 methods where you are calling them from to keep the mailers as conventional as possible.

Reply:
Will move, just to explain my reasoning: I put them here so as not to expose the internals of the mailer. I wanted to encapsulate all the notify weirdness, so there's only one place to change if we change the notify gem